src/crc128/default.rs

```diff
@@ -1,11 +1,44 @@
-use crate::table::crc128_table;
-use crate::{Algorithm, Crc, Digest};
+use crate::{Algorithm, Crc, Digest, Implementation};
 
-use super::{finalize, init, update_bytewise};
+use super::{finalize, init};
+
+#[cfg(feature = "notable-defaults")]
+impl Implementation for u128 {
+    type Width = u128;
+    type Table = ();
+}
+
+#[cfg(all(not(feature = "notable-defaults"), feature = "bytewise-defaults"))]
+impl Implementation for u128 {
+    type Width = u128;
+    type Table = [u128; 256];
+}
+
+#[cfg(all(
+    not(feature = "notable-defaults"),
+    not(feature = "bytewise-defaults"),
+    feature = "slice16-defaults"
+))]
+impl Implementation for u128 {
+    type Width = u128;
+    type Table = [[u128; 256]; 16];
+}
 
 impl Crc<u128> {
     pub const fn new(algorithm: &'static Algorithm<u128>) -> Self {
```

Comment:
This fails to compile if the crate is imported with `default-features = false`. I think `bytewise` should be used as the default in that case.

I know some people always import dependencies with `default-features = false` even if the crate doesn't actually make use of features.